### PR TITLE
changes for selective picking of destinations

### PR DESCRIPTION
**Pull request by sayan-rudder**
**Size:         29.05 KB with all dependencies, minified and brotli
  Loading time: 581 ms   on slow 3G
  Running time: 228 ms   on Snapdragon 410
  Total time:   809 ms**
  

This change is [Reviewable](https://reviewable.io/reviews/rudderlabs/rudder-sdk-js/65)

### CODE REVIEW

File: utils/utils.js [hunk 1]
```diff
@@ -212,6 +213,91 @@ function getRevenue(properties, eventName) {
   return getCurrency(revenue);
 }
 
+/**
+ *
+ *
+ * @param {*} integrationObject
+ */
+function tranformToRudderNames(integrationObject) {
+  Object.keys(integrationObject).forEach(key => {
```
Comment (prabrisha-rudder) [inline utils/utils.js:222]: should we make this search case insensitive?

File: utils/utils.js [hunk 2]
```diff
@@ -212,6 +213,91 @@ function getRevenue(properties, eventName) {
...
+  }
+  let allValue = true
+  if(typeof configPlaneEnabledIntegrations[0] == "string") {
+    if(sdkSuppliedIntegrations["All"] != undefined) {
+      allValue = sdkSuppliedIntegrations["All"]
+    }
+    configPlaneEnabledIntegrations.forEach(intg => {
+      if(!allValue) {
+        // All false ==> check if intg true supplied
+        if(sdkSuppliedIntegrations[intg] && sdkSuppliedIntegrations[intg]) {
```
Comment (prabrisha-rudder) [inline utils/utils.js:254]: check once more

Comment (sayan-rudder) [reply utils/utils.js]: done

File: utils/utils.js (hunk 1 above)
Comment (sayan-rudder) [reply utils/utils.js:223]: skipping as discussed

**Pull request merged**